frontend/src/common/enums/ui/button-color.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum ButtonColor {
```

Comment:
```suggestion
enum ButtonStyle {
```
don't forget about the file name